//test exception 8

int main() {
    int *x = NULL;
    *x = 1 / 0;
    return 0;
}

